### ISSUE TITLE
Which platform was onnc tested when collaborated with NVDLA

### ISSUE THREAD

**Issue by LeiWang1999** [ONNC/onnc#191]
Hi, I notice that onnc just can support nvdla full version.

But the full version of nvdla was too large to mapping into a FPGA Platform, so when I use nvdla backend to generate a loadable but how can I evaluate this loadable can work correctly？

DId ONNC use the QEMU to evaluate this? If so,  It is impossile to debug the runtime code, and the runtime was too long.

**Comment by RaphaelKlink** [comment]
Yes i have the same problem. I searched through the files. You can change the used ressources, but you can not change the support modules. So currently i do not see a possibility without buying the commercial license
